Core.IntegrationTests/Jira/JiraCombinedIntegrationTests.cs

```diff
@@ -422,32 +422,27 @@ public void ReleaseVersionAndMoveIssues_MovesProperIssues ()
     var jiraVersionRepairer = new JiraProjectVersionRepairer(_service, _versionFinder);
     var jiraVersionReleaser = new JiraVersionReleaser(_config, jiraVersionRepairer, _service);
 
-    var releaseVersionAndMoveIssues = new ReleaseVersionAndMoveIssuesSubStep(
-        testConsole,
-        inputReader.Object,
-        _issueService,
-        jiraVersionCreator,
-        jiraVersionReleaser);
+    var releaseVersionAndMoveIssues = new ReleaseVersionAndMoveIssuesSubStep(_config, testConsole, inputReader.Object, _issueService, jiraVersionCreator, jiraVersionReleaser);
```

Comment:
Probably auto-formatted but we should keep the multi-line paramters. Other such case down below as well.